Connections/Shear/Endplate/endPlateCalc.py

```diff
@@ -232,36 +248,77 @@ def end_connection(ui_obj):
     
     
 # ############# BOLT CAPACITY ###############
-    
-    # I: Check for number of bolts -------------------
-    
+#    0 def boltDesign(end_plate_l):
+# I: Check for number of bolts -------------------
+
     bolt_fu = int(bolt_grade) * 100
     bolt_fy = (bolt_grade - int(bolt_grade)) * bolt_fu
-    
+
     if connectivity == "Column web-Beam web":
         t_thinner = min(column_w_t.real, end_plate_t.real)
     elif connectivity == "Column flange-Beam web":
         t_thinner = min(column_f_t.real, end_plate_t.real)
     else:
         t_thinner = min(column_w_t.real, end_plate_t.real)
-    
+
+    # Spacing of bolts for web plate -------------------
+    # ######## According to IS 800 - 2007, table 9, clause no. 10.2.1 ##########################
+
```

Comment:
The following checks for distances could go in a separate function, to enhance readability of code.